### ISSUE TITLE
jsonld through rdflib vs on its own

### ISSUE THREAD

**Issue by satra** [trungdong/prov#87]
@trungdong - is there a reason why you created a separate jsonld branch instead of trying it through the rdf branch?

**Comment by trungdong** [comment]
During an email conversation with @cmaumet, she asked whether we support JSON-LD export. I created the branch to share the code I developed earlier this year for this with her.

The JSON-LD export code generates JSON-LD directly without going via a RDF graph. We originally developed it to address the issue of PROV-JSON being not streaming-friendly (due to its highly-nested objects). In our JSON-LD representation, each PROV record is serialised into one JSON object, which can be deserialised on its own (in conjunction with the active JSON-LD context). Hence, each record can be deserialised immediately once its JSON object has been completely read. A normal JSON-LD export does not guarantee this.
